src/test/groovy/com/coxautodev/graphql/tools/TypeClassMatcherSpec.groovy

```diff
@@ -79,21 +79,6 @@ class TypeClassMatcherSpec extends Specification {
             "futureType"   | list()
     }
 
-    @Unroll
-    def "matcher verifies return value optionals are used incorrectly for method #methodName"() {
-        when:
-            matcher.match(createPotentialMatch(methodName, type))
-
-        then:
-            thrown(SchemaClassScannerError)
-
-        where:
-            methodName             | type
-            "nullableType"         | nonNull(customType)
-            "nullableNullableType" | customType
-            "listNullableType"     | list(customType)
-    }
-
```

Comment:
Same here, I may be wrong, but I'm not sure this test makes sense anymore. Haven't added any test for `transformer`, as `Optional`s are already tested and are now handled via a `GenericWrapper`.